### ISSUE TITLE
Get rid of npmconf

### ISSUE THREAD

**Issue by jakubzitny** [giggio/node-chromedriver#45]
Hey, you might wanna get rid of the `npmconf` dependency.

NPM throws this warning:

```
npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm
```

If interested, I can submit a PR for replacing the functionality.

**Comment by gilligan** [comment]
Nothing much going on here. @jakubzitny maybe just go ahead and create a PR and then maybe if enough people vote for it we can get it merged?

**Comment by giggio** [comment]
@jakubzitny sure, send a PR!

**Comment by aberonni** [comment]
Any update on this? Not sure how complicated it could be - I might look into this myself if nobody else is looking into it.

**Comment by jakubzitny** [comment]
@aberonni do it if you have some time.

**Comment by protron** [comment]
I haven't tried myself, but I found that others ([node-sass](https://github.com/sass/node-sass/pull/1413/commits/5e21f47576b9d75859db71512decbaa9fc3ffca2) & [phantomjs](https://github.com/Medium/phantomjs/commit/2620fb283439181cab0a99b89fdf4532455bc35d#diff-f16acefe4b6553580c43edab685f50f3L168)) are now just getting these settings from the process environment variables: `process.env.npm_config_https_proxy || process.env.npm_config_http_proxy || process.env.npm_config_proxy`.

**Comment by giggio** [comment]
Fixed and published as `2.27.1.
